### DISCUSSION TITLE
Please verify that the --pid_suffix option works as intended

### DISCUSSION THREAD

**Discussion by ghost** [gaogaotiantian/viztracer#345] [General]
Recent versions of viztracer don't seem to append the PID to the filename. Instead it creates `result.json`.

`viztracer --pid_suffix single_process.py` appends the PID in version 0.12.3 but not in 0.15.6

https://viztracer.readthedocs.io/en/latest/concurrency.html#combine-reports

**Comment by gaogaotiantian** [comment]
This is a regression and the issue is created at #346 . I'll fix it soon, thanks!

**Reply by ghost** [reply]
Thank you, do you think the example `viztracer --pid_suffix single_process.py` should work now?
I'm on the latest commit but no PID is appended to the filename still.
https://github.com/gaogaotiantian/viztracer/pull/348
